share/revng/test/tests/invalidation/invalidation.c.override.yml

```diff
@@ -0,0 +1,164 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+---
+Functions:
+  - CustomName: "bar"
+    OriginalName: bar
+    Prototype: "/Types/RawFunctionType-13090241523659220176"
+  - CustomName: "foo"
+    OriginalName: foo
+    Prototype: "/Types/RawFunctionType-15357695279870196297"
+  - CustomName: "qux"
+    OriginalName: qux
+    Prototype: "/Types/RawFunctionType-6494433326490644482"
+  - CustomName: "baz"
+    OriginalName: baz
+    Prototype: "/Types/RawFunctionType-12005182500180626567"
+  - CustomName: "main"
+    OriginalName: main
+    Prototype: "/Types/CABIFunctionType-17140048229552705196"
+ImportedDynamicFunctions:
+  - OriginalName: __libc_start_main
+    CustomName: "prefix___libc_start_main"
+    Prototype: "/Types/CABIFunctionType-3475954398804502039"
+    Relocations:
+      - Address: "0x402000:Generic64"
```

Comment:
* In tests we never ever commit hard-coded addresses.
* Drop all the stuff that you didn't write by hand such as `prefix___libc_start_main`